docs/reference/migration/migrate_7_12.asciidoc

```diff
@@ -0,0 +1,33 @@
+[[breaking-changes-7.12]]
+== Breaking changes in 7.12
+++++
+<titleabbrev>7.12</titleabbrev>
+++++
+
+This section discusses the changes that you need to be aware of when migrating
+your application to {es} 7.12.
+
+See also <<release-highlights>> and <<es-release-notes>>.
+
+// * <<breaking_712_blah_changes>>
+// * <<breaking_712_blah_changes>>
+
+//NOTE: The notable-breaking-changes tagged regions are re-used in the
+//Installation and Upgrade Guide
+
+//tag::notable-breaking-changes[]
+
+[discrete]
+[[breaking_712_engine_changes]]
+=== Engine changes
+
+[[breaking_712_engine_forcemerge_change]]
+.Force-merges on frozen and searchable snapshot indices will fail if merging is required
+[%collapsible]
+====
+*Details* +
+In earlier versions a force-merge on a frozen index or a searchable snapshot
+index would yield a successful response without performing the requested merge.
+From version 7.12 onwards a force-merge on these kinds of index will fail
```

Comment:
I think we should frame this as a bug, something like:

`This bug has been fixed in version 7.12 onwards such that a force-merge ...`